test.py

```diff
@@ -0,0 +1,6 @@
+def main():
```

Comment:
I'm assuming this is required in order to test flake8?

Reply:
Yes.